docs/architecture.md

```diff
@@ -33,4 +33,25 @@ An analyzer is a gRPC service that will be called by the [Server](#server) to pe
 
 They are not part of **source{d} Lookout** repository so they can be developed by third parties.
 
-source{d} Lookout Server will call all the registered Analyzers to produce comments for the opened Pull Request in the watched repositories. To register new Analyzers in the configuration file, `lookoutd` will need to be restarted.
+**source{d} Lookout** Server will call all the registered Analyzers to produce comments for the opened Pull Request in the watched repositories. To register new Analyzers in the configuration file, `lookoutd` will need to be restarted.
+
+
+# External services
+
+**source{d} Lookout** will call some external services, some of them depending on **source{d} Lookout** configuration or on how it is run.
+
+## Babelfish
+
+Babelfish is used to parse files into [UAST](https://doc.bblf.sh/uast/uast-specification.html) when the [DataService](#dataservice) `GetChanges` or `GetFiles` methods are asked to include UASTs.
+
+Babelfish can also be directly called through **DataService** proxy.
```

Comment:
Do we have a link for the babelfish proxy methods?

Reply:
I don't understand this suggestion. Link to source code? bblfsh docs? link to docs explaining the methods that have been proxied and how to use them?

Reply:
I was asking about some section we could link. I see we mention it in this file, in the DataService section, but I wanted to know if there is some other place where we define with more detail what methods can you use, an example or something. Because I think we don't have a more detailed explanation in the docs, but I wanted to ask if you know if we have something like that and maybe I didn't see it.

If the only place where we mention it is this file, then I don't think we need a link.

Reply:
We don't have such explanation.
If you agree, I can create an issue to require new docs about how to use DataService. We could add examples about how to call it and what data to extract, and also a call to bblfsh using DataService proxy.
Or do you think that current examples in `lookout-sdk/examples/language-analyzer.*` is enough?

Reply:
let's create an issue and decide later the specifics :+1: